app/charts/map/map-state.tsx

```diff
@@ -96,22 +96,24 @@ const getColorScale = ({
   dataDomain: [number, number];
   nbClass: number;
 }) => {
-  const paletteDomain = getSingleHueSequentialPalette({
-    palette,
-    nbClass: 9,
-  });
+  const interpolator = getColorInterpolator(palette);
+  const getDiscreteRange = () => {
+    return Array.from({ length: nbClass }, (_, i) =>
+      interpolator(i / (nbClass - 1))
+    );
+  };
```

Comment:
TIL that array from can take a function map as 2nd argument 👍 I would always map in the past to achieve the same effect.

Reply:
Yeah, I'm not sure if it's more performant, but for sure it's nice :)